acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -75,4 +82,24 @@
   on agent, "#{ruby} --version"
 end
 
+agents.each do |agent|
+  on agent, puppet('--version')
+  ruby = Puppet::Acceptance::CommandUtils.ruby_command(agent)
+  on agent, "#{ruby} --version"
+end
+
+# Get a rough estimate of clock skew among hosts
+times = []
+hosts.each do |host|
+  ruby = Puppet::Acceptance::CommandUtils.ruby_command(host)
+  on(host, "#{ruby} -e 'puts Time.now.strftime(\"%Y-%m-%d %T.%L %z\")'") do |result|
+    times << result.stdout.chomp
+  end
+end
+times.map! do |time|
+  (Time.strptime(time, "%Y-%m-%d %T.%L %z").to_f * 1000.0).to_i
+end
+diff = times.max - times.min
+logger.warn "Host times vary #{diff} ms"
```

Comment:
@er0ck Using `logger.warn` seems to work, but I'm not sure if that's beaker best practice